docs/monitors/elasticsearch.md

```diff
@@ -35,7 +35,7 @@ Monitor Type: `elasticsearch`
 | `enableEnhancedClusterHealthStats` | no | `bool` | Enable Cluster level stats. These stats report only from master Elasticserach nodes (**default:** `false`) |
 | `indexStatsIntervalSeconds` | no | `integer` | Interval to report IndexStats on (defaults to 60s) (**default:** `60`) |
 | `indexSummaryOnly` | no | `bool` | Collect only aggregated index stats across all indexes (**default:** `false`) |
-| `indexStatsMasterOnly` | no | `bool` | Collect index stats only from Master node. By default the monitor will collect these from master eligible nodes. (**default:** `false`) |
+| `indexStatsMasterOnly` | no | `bool` | Collect index stats only from Master node (defaults to true) (**default:** `true`) |
```

Comment:
You don't need to say `(defaults to true)`, the doc generator will add it for you.